.travis.yml

```diff
@@ -2,7 +2,7 @@ language: ruby
 
 services: memcached
 
-rvm: 2.2.1
```

Comment:
Yep, forgot to upgrade Travis in the original commit, so [this build](https://travis-ci.org/bundler/bundler-api/builds/59629929) is expected to fail (as it did).